repository/kafka/docs/latest/release-notes.md

```diff
@@ -0,0 +1,5 @@
+
+- Apache Kafka upgraded to 2.3.0.
```

Comment:
Should you put a date?

Reply:
When releasing yes, right now its in `latest` directory. So when doing the release the directory should moved to its versioned name directory and add the date also